lib/pwa.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Progressive Web App
+ *
+ * @package gutenberg
+ */
+
+add_filter(
+	'admin_head',
+	function() {
+		$l10n = array(
+			'logo'      => file_get_contents( ABSPATH . 'wp-admin/images/wordpress-logo-white.svg' ),
```

Comment:
Should this incorporate the site icon? Otherwise a user may have multiple admin PWAs and they'd all have the same icon.

Reply:
I've thought a bit about this, and I think we shouldn't use the site icon. See https://github.com/WordPress/gutenberg/pull/33102#issuecomment-872887040.

I was using the site icon at first, but it only works with pngs in Chrome, and the front-end of the site could potentially be a PWA that uses the site icon. The site icon is meant to be used for the site, not the administration area. I've made it so that the admin icon is dynamically generated based on the admin color scheme, so you could have many different WordPress sites that you're managing with differently coloured icons. I think this is the nicest solution for the admin, for which we'd need a default icon anyway if so site icon is set.

Reply:
Example:

![image](https://user-images.githubusercontent.com/4710635/124722635-e76cba80-df12-11eb-9c70-bdf1be09c5f5.png)

Reply:
How often do users change the admin color scheme? I personally never change it. It's the same for all sites I manage.

Reply:
> The site icon is meant to be used for the site, not the administration area.

Also, I'm not sure about that. Is not the site icon also showing up as the favicon when in the admin?